app/views/jobseekers/job_applications/build/personal_statement.html.slim

```diff
@@ -8,6 +8,20 @@
     h1.govuk-heading-l = t(".heading")
     p.govuk-body = t(".description")
 
+    - if job_application.vacancy.documents.any?
+      .govuk-notification-banner aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner" role="region"
```

Comment:
we should be able to use `govuk_notification_component` (or it's called something like that)